core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -2598,4 +2654,40 @@ public static SqlOperator reverse(SqlOperator operator) {
     }
   }
 
+  /**
+   * The mapping from a SQL operator to the libraries it belongs to.
+   */
+  static Map<SqlOperator, Set<SqlLibrary>> operatorLibraries = new HashMap<>();
+
```

Comment:
This mapping cannot be static. We don't know the full set of operator tables.

Reply:
Sounds reasonable. Revised accordingly. Thanks for your feedback.